ibeam/src/gateway_client.py

```diff
@@ -67,6 +69,7 @@ def __init__(self,
         self.two_fa_handler = two_fa_handler
 
         self._concurrent_maintenance_attempts = 1
+        self._health_server = new_health_server(IBEAM_HEALTH_SERVER_PORT, self.get_status, self.get_shutdown_status)
```

Comment:
For consistency, let's use `var.IBEAM_HEALTH_SERVER_PORT` and remove the bespoke import of this env var

Reply:
Fixed